07-13-16-language-plugins.md

```diff
@@ -0,0 +1,94 @@
+## A Milestone for Tessel
+This week we merged one of our most significant pull requests: [adding Rust deployment support](https://github.com/tessel/t2-cli/pull/790). For the past three years we've focused exclusively on deploying JavaScript on microcontrollers but this is our first step into a new language and a new community. We're not leaving JavaScript behind, rather, we see a future where the languages compliment each other (more on that in a subsequent post).
+
+Despite the significance of this addition to the CLI codebase, the amount of code flux was relatively small. We basically added two files: [one with the implementation](https://github.com/tessel/t2-cli/blob/b8cd1f34398c2d188fa188a216b4b9b4dd206dc4/lib/tessel/deployment/rust.js) and one with [unit tests](https://github.com/tessel/t2-cli/blob/b8cd1f34398c2d188fa188a216b4b9b4dd206dc4/test/unit/deployment/rust.js). That's a very good thing and it's largely due to the hardwork of [@rwaldron](https://github.com/rwaldron). In this post, I want to demonstrate how simple it is to add support for a new language to the CLI using the T2 language plugin system.
```

Comment:
Yeah, I can see why the first paragraph is misleading. I'm editing it to stay more on topic about new languages in general, instead of specifically Rust.